cocoa/scheduler.py

```diff
@@ -27,6 +47,8 @@ def reschedule(self, uid: int, user: User, channel: TextChannel, client: Client)
         scheduled_meetings = self.modelsvc.get_meetings(uid=uid)
         self.notifiersvc.notify_reschedule(scheduled_meetings, user, channel, client)
 
-    # List all meetings
     def list_booked_meetings(self, user: User, channel: TextChannel):
-        self.notifiersvc.notify_multiple_meetings(user, channel)
\ No newline at end of file
+        self.notifiersvc.notify_multiple_meetings(user, channel)
+
+    def delete_user(self, author):
```

Comment:
Hey @AirinB, ```delete_user``` won't be there in ```scheduler.py```. Actually, it was a mistake on my part. Do 1 thing, go to ```controller.py```, and in line 27, change **schedulersvc** to **configsvc**.
And remove this method from this file.